common/app/routes/Challenges/views/invalid/NotFound.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from 'react';
+
+// import PropTypes from 'prop-types';
+/* eslint max-len: ["error", { "ignoreStrings": true }] */
+
+
+const propTypes = {};
+
+export class NotFound extends PureComponent {
+  render() {
+    return (
+        <div className='container'>
+          <div className='row'>
+            <div className='col-sm'>
+            <div className='alert alert-danger' role='alert'>
+              <h4 className='alert-heading' className='mb-12'>
+                This challenge is not found.</h4>
+               <p>Head back to <a className='alert-link'
```

Comment:
We have a `<Link />` component to handle link navigation, It is available in `common/app/Router`

```jsx
<Link
  to='/challenges/current-challenge'
/>
```